i18n: update Portuguese translation

ivangomes opened the pull request:
@xyloflake  & @staniel359 See the Brazilian Portuguese translations. Let me know if any adjustments are needed.

ivangomes committed:
i18n: update Brazilian portuguese translation

ivangomes committed:
i18n: update Brazilian portuguese translation

staniel359 commented on electron/plugins/i18n/locales/br.json:
```diff
@@ -0,0 +1,13 @@
+      "close": "Fechar"
+    }
+  }
+}
```
Should be new line here

staniel359 commented on src/plugins/i18n/locales/br.json:
```diff
@@ -0,0 +1,952 @@
+{
```
The indentation should be 2 spaces, not 4.

staniel359 commented on src/plugins/i18n/locales/br.json:
```diff
@@ -0,0 +1,952 @@
+    "more": "Mais...",
+    "loading": "Carregando...",
+    "warning": "Aviso"
+}
```
New line here too.

staniel359 reviewed (changes requested):
@ivangomes Please also update `src/plugins/dayjs.js`

xyloflake commented:
@staniel359 please let me know when we are ready to provide a private beta for testing purposes. I have gone ahead and set up the CI/CD for the beta rollout.

ivangomes committed:
indentation 2 spaces and new line added

ivangomes committed:
indentation 2 spaces and new line added

staniel359 commented:
@xyloflake Please trigger a beta rollout.

xyloflake commented:
@staniel359 on it

xyloflake commented:
https://github.com/muffonlabs/mufbeta/releases/tag/v2.3.0-beta.1

@ivangomes please test the release.

ivangomes committed:
using portugese per ISO 639-1

ivangomes committed:
using portugese per ISO 639-1

staniel359 commented on electron/plugins/i18n.js:
```diff
@@ -10,6 +10,7 @@ import it from './i18n/locales/it.json' with { type: 'json' }
 import ja from './i18n/locales/ja.json' with { type: 'json' }
 import ko from './i18n/locales/ko.json' with { type: 'json' }
 import pl from './i18n/locales/pl.json' with { type: 'json' }
+import pt from './i18n/locales/pt.json' with { type: 'json' }
```
@ivangomes So is this the Brazilian Portuguese or the Portuguese itself?

xyloflake replied:
It's brazilian portugese

staniel359 replied:
@ivangomes Please rename to `pt_BR`

staniel359 commented on src/plugins/dayjs.js:
```diff
@@ -13,6 +13,7 @@ import 'dayjs/locale/it'
 import 'dayjs/locale/ja'
 import 'dayjs/locale/ko'
 import 'dayjs/locale/pl'
+import 'dayjs/locale/pt'
```
@ivangomes Please change to `pt-br`

staniel359 commented on electron/plugins/i18n/locales/pt.json:
```diff
@@ -0,0 +1,13 @@
+{
```
@ivangomes Please rename to `pt_BR.json`

staniel359 commented on src/components/BaseEmojiPicker.vue:
```diff
@@ -23,6 +23,7 @@ import fr from 'emoji-picker-element/i18n/fr'
 import it from 'emoji-picker-element/i18n/it'
 import ja from 'emoji-picker-element/i18n/ja'
 import pl from 'emoji-picker-element/i18n/pl'
+import pt from 'emoji-picker-element/i18n/pt_BR'
```
@ivangomes Please rename to `pt_BR`

staniel359 commented on src/helpers/data/locales.js:
```diff
@@ -42,6 +42,10 @@ export default [
     name: 'Polski'
   },
+  {
+    id: 'pt',
```
@ivangomes Please rename to `pt_BR`

staniel359 commented on src/helpers/data/locales.js:
```diff
@@ -42,6 +42,10 @@ export default [
   },
+  {
+    id: 'pt',
+    name: 'Português'
```
@ivangomes Please change to Brazilian variant

staniel359 commented on src/plugins/i18n.js:
```diff
@@ -11,6 +11,7 @@ import it from './i18n/locales/it.json'
 import ja from './i18n/locales/ja.json'
 import ko from './i18n/locales/ko.json'
 import pl from './i18n/locales/pl.json'
+import pt from './i18n/locales/pt.json'
```
@ivangomes Please rename to `pt_BR`

staniel359 commented on src/plugins/i18n/locales/pt.json:
```diff
@@ -0,0 +1,952 @@
+{
```
@ivangomes Please rename to `pt_BR.json`

staniel359 commented on src/plugins/i18nCountries.js:
```diff
@@ -11,6 +11,7 @@ import it from 'i18n-iso-countries/langs/it.json'
 import ja from 'i18n-iso-countries/langs/ja.json'
 import ko from 'i18n-iso-countries/langs/ko.json'
 import pl from 'i18n-iso-countries/langs/pl.json'
+import pt from 'i18n-iso-countries/langs/pt.json'
```
@ivangomes Please create a file
`src/plugins/i18nCountries/locales/pt_BR.json`
with contents of [this file](https://github.com/michaelwittig/node-i18n-iso-countries/blob/master/langs/pt.json)
and import it here like this:
`import pt_BR from './i18nCountries/locales/pt_BR.json`

ivangomes committed:
Add Brazilian Portuguese localization for countries

- Replaced the Portuguese localization with Brazilian Portuguese (pt_BR).
- Added a new JSON file for Brazilian Portuguese translations of country names.
- Updated the i18nCountries.js file to include the new pt_BR localization.

ivangomes committed:
Merge branch 'main' of https://github.com/ivangomes/muffon into main

ivangomes committed:
remove portuguese (Portugal)

ivangomes committed:
Merge branch 'staniel359:main' into main

staniel359 commented on electron/plugins/i18n.js:
```diff
@@ -10,6 +10,8 @@ import it from './i18n/locales/it.json' with { type: 'json' }
 import ja from './i18n/locales/ja.json' with { type: 'json' }
 import ko from './i18n/locales/ko.json' with { type: 'json' }
 import pl from './i18n/locales/pl.json' with { type: 'json' }
+// eslint-disable-next-line camelcase
```
@ivangomes Here and later:
if there's a problem with ESLInt, please change similar imports to `import ptBr`.
But keep file names as `pt_BR`.
And don't forget to remove this ESLint string.

ivangomes replied:
@staniel359 Done!

ivangomes committed:
eslint camelcase

staniel359 commented on src/plugins/i18n.js:
```diff
@@ -11,6 +11,8 @@ import it from './i18n/locales/it.json'
 import ja from './i18n/locales/ja.json'
 import ko from './i18n/locales/ko.json'
 import pl from './i18n/locales/pl.json'
+// eslint-disable-next-line camelcase
```
@ivangomes Forgot these lines.

staniel359 commented on src/plugins/i18n.js:
```diff
@@ -29,6 +31,8 @@ const localesData = {
   ja,
   ko,
   pl,
+  // eslint-disable-next-line camelcase
```
@ivangomes Here too.

staniel359 commented on src/plugins/i18nCountries.js:
```diff
@@ -11,6 +11,8 @@ import it from 'i18n-iso-countries/langs/it.json'
 import ja from 'i18n-iso-countries/langs/ja.json'
 import ko from 'i18n-iso-countries/langs/ko.json'
 import pl from 'i18n-iso-countries/langs/pl.json'
+// eslint-disable-next-line camelcase
```
@ivangomes Here and below too.

ivangomes committed:
eslint camelcase i18n ptBr

gamersi reviewed (approved):
Good Job!

xyloflake reviewed (approved):
LGTM!
